### PR TITLE
Update Transition docs

### PR DESCRIPTION
**Pull request by Rosa-Fox**
[Trello](https://trello.com/c/Rzil6Irn/74-consolidate-transition-and-bouncer-documentation)

### CODE REVIEW

File: source/manual/bouncer.html.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+which adds and removes domains to and from the service.
+That job fetches the list of domains which should be configured at the CDN from
+Transition's [hosts API](https://transition.publishing.service.gov.uk/hosts), so
+will fail if that is unavailable.
+
+[More information about Bouncer's Fastly service](https://docs.publishing.service.gov.uk/manual/cdn.html#bouncer39s-fastly-service)
+
+### Machines
+
+Bouncer runs on 3 machines in the `redirector` vDC (`bouncer-[1-3].redirector`),
```
Comment (jennyd) [inline source/manual/bouncer.html.md]: It's probably worth mentioning the DR machines and automatic fallback here too.

File: source/manual/bouncer.html.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+title: Bouncer
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+old_path_in_opsmanual: "../opsmanual/2nd-line/applications/bouncer-and-transition.html"
+---
+
+# Bouncer
+
+[Bouncer](https://github.com/alphagov/bouncer) is a Ruby/Rack web app that receives requests for the old URLs,
```
Comment (36degrees) [inline source/manual/bouncer.html.md]: I'm not sure it's clear what 'old URLs' refers to here – can this be a little more specific?

File: source/manual/bouncer.html.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+### DNS
+
+When sites transition they are generally CNAMEd to a domain we control that
+points to our CDN (an A record is used for root domains which can't be CNAMEd).
+
+Some sites partially transition, which means that they redirect some paths to
+their AKA domain, which is CNAMEd to us.
+
+GDS doesn't control the DNS for most transitioned domains, except for
+`*.direct.gov.uk`, `*.businesslink.co.uk`, `*.alphagov.co.uk` etc. If the DNS
```
Comment (36degrees) [inline source/manual/bouncer.html.md]: The use of etc is confusing here. Is this is an exhaustive list or not?

File: source/manual/data-sources-for-transition.html.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# Data sources for Transition
+
+Site configuration is automatically imported on an hourly basis via
+[deploy.publishing](https://deploy.publishing.service.gov.uk/job/Transition_load_site_config/) from
+[transition-config](https://github.com/alphagov/transition-config).
+
+Pre-transition traffic data is imported from
+[pre-transition-stats](https://github.com/alphagov/pre-transition-stats),
+based on logs provided by transitioning organisations. This was updated
```
Comment (36degrees) [inline source/manual/data-sources-for-transition.html.md]: Has the whole thing (importing traffic data) come to an end, or just the updating by hand? If this is historical, suggest rewriting the whole paragraph so it's in the past tense.

Comment (jennyd) [inline source/manual/data-sources-for-transition.html.md]: Importing data for sites before they transition happens on a one-off basis - if we decide we need to do this for a particular site, we'd get logfiles from the org, process them manually, commit them to this repo and they would get imported - that only needs to happen once.

We did this for local.direct.gov.uk last year, and discussed doing it for at least one of the upcoming PHE transitions, and it should be considered as an option for other future ones, even if we only do it rarely.

**Comment by Rosa-Fox** [comment]
Thanks @36degrees and @jennyd. I have made the changes, let me know if you have any other suggestions.

**Pull request merged**